### ISSUE TITLE
Is this docker image up-to-date?

### ISSUE THREAD

**Issue by AhmedMozaly** [ripple/docker-fdb-server#1]
As Apple open-sourced FDB (https://github.com/apple/foundationdb)
May I know if this docker image contains last version of FDB currently available (April 2018) ?

**Comment by emschwartz** [comment]
No, sorry, this is a very old image that was put up before Apple bought the company
